### PR TITLE
Spotlights of September

### PR DESCRIPTION
**Pull request by Shadowthegod**
### Description
Article of spotlights
### Status
Pending Review

### CODE REVIEW

File: news/2017-10-28-beatmap-spotlights-september-2017.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+ 
+Don’t discount the rest of the spread though, especially not the ‘Collab Insane’, which consists of varied unique patterns and interesting stylistic choices! With such diverse talent, this is one gem you certainly don’t want to miss!
+
+[![](/wiki/shared/news/2017-10-28-beatmap-spotlights-september-2017/chrono-diver.jpg)](https://osu.ppy.sh/s/419600)
+
+**[toybot](https://osu.ppy.sh/u/2848604)**, [ProfessionalBox](https://osu.ppy.sh/u/3280792), [Sing](https://osu.ppy.sh/u/3795679) & [byfar](https://osu.ppy.sh/u/4674054). A quick glance at the collaborator's names would already tell you that you're in for an extravagant composition. Filled with modern patterns and energy-filled gameplay, this mapset is accompanied by an alluring storyboard put together by famous storyboarders [-Tochi](https://osu.ppy.sh/u/3664366) and [Storyboarder](https://osu.ppy.sh/u/6124671)!
```
Comment (Xinnoh) [inline news/2017-10-28-beatmap-spotlights-september-2017.md]: The image should probably include byfar, probox and toybot too

File: news/2017-10-28-beatmap-spotlights-september-2017.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+[![](/wiki/shared/news/2017-10-28-beatmap-spotlights-september-2017/magical-sanctuary.jpg)](https://osu.ppy.sh/s/534568)
+
+Just two months after the very first ranked Kobaryo map and already we are treated to another! Consisting of 7 difficulties mapped by [Raiden](https://osu.ppy.sh/u/2239480) himself, Magical Sanctuary is an excellent addition to the rising popularity of speedcore maps in osu!taiko.
+
+Rushing past at 333 BPM (though mapped to 166.5), the map is an exciting rush of adrenaline for all players. In the hardest difficulty "Hakai", Raiden does not shy away from using slider velocity changes and more complex snaps to further emphasize the intense nature of the song. A worthy addition to the monthly spotlights for sure.
+
+[![](/wiki/shared/news/2017-10-28-beatmap-spotlights-september-2017/maniera.jpg)](https://osu.ppy.sh/s/400761)
```
Comment (Xinnoh) [inline news/2017-10-28-beatmap-spotlights-september-2017.md]: should use commas for the first three, &&& not needed

File: news/2017-10-28-beatmap-spotlights-september-2017.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+[![](/wiki/shared/news/2017-10-28-beatmap-spotlights-september-2017/lilieze-to-enryuu-laevateinn.jpg)](https://osu.ppy.sh/s/415827)
+
+Rounding out this month's spotlights, [Kyuare](https://osu.ppy.sh/u/2705312)'s mapset of 'Lilieze to Enryuu Laevateinn' brings us another taste of their signature style. Rapid direction changes and dense streams requiring rapid inputs provide a consistently challenging flow but still feels natural to play. With a healthy spread of difficulties this mapset is worth checking out for any catch player.
+
+## osu!mania
+
+[![](/wiki/shared/news/2017-10-28-beatmap-spotlights-september-2017/airborne-robots.jpg)](https://osu.ppy.sh/s/473736)
+
+I remember back then when [Curiossity](https://osu.ppy.sh/u/6490035) asked me to mod this back when I wasn't even part of the BNG and the map was overall way too repetitive and copy-pasted but now, after a looong time and a good amount of modding I am sure we can all agree this is one of the best maps that 4K has seen this year.
```
Comment (Xinnoh) [inline news/2017-10-28-beatmap-spotlights-september-2017.md]: this is too casual, don't think I or me should be used in spotlights

File: news/2017-10-28-beatmap-spotlights-september-2017.md [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+
+Starting the map, the player is introduced to some light jacks with half a measure long hold notes, which leads into a buildup of 1/8 bursts that will challenge the player’s speed and reading. Making it past that is a calm section consisting of many half-beat Long Notes of which goes into a transitional phrase with a fair amount of minijacks peppered throughout.
+
+Players will be taken by pleasant surprise at the height of the song with a long note section full of quickly timed 1/4ths which will require precise holding and releasing to get through with a high accuracy. While not terribly difficult, this will definitely challenge and force your average player out of their comfort zone and serves as an excellent introduction to harder types of patterning not commonly found in the ranked section.
+
+--------
+
+And that is the end of this month's Spotlight listing. We hope that you click, drum, catch, and mash your heart out from these maps and earn a shiny medal for your profile page! Look forward to next month's installment!
```
Comment (Xinnoh) [inline news/2017-10-28-beatmap-spotlights-september-2017.md]: out from these -> out on these

File: news/2017-10-28-beatmap-spotlights-september-2017.md [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+
+[![](/wiki/shared/news/2017-10-28-beatmap-spotlights-september-2017/yell.jpg)](https://osu.ppy.sh/s/591786)
+
+Mania’s had a rather lackluster month this time around, so [Maxus](https://osu.ppy.sh/u/4335785)’s newest marathon map YELL!, with long notes, slider velocity changes, and jacks is a refreshing addition to this month’s spotlights.
+
+Starting the map, the player is introduced to some light jacks with half a measure long hold notes, which leads into a buildup of 1/8 bursts that will challenge the player’s speed and reading. Making it past that is a calm section consisting of many half-beat Long Notes of which goes into a transitional phrase with a fair amount of minijacks peppered throughout.
+
+Players will be taken by pleasant surprise at the height of the song with a long note section full of quickly timed 1/4ths which will require precise holding and releasing to get through with a high accuracy. While not terribly difficult, this will definitely challenge and force your average player out of their comfort zone and serves as an excellent introduction to harder types of patterning not commonly found in the ranked section.
```
Comment (Xinnoh) [inline news/2017-10-28-beatmap-spotlights-september-2017.md]: Is it pleasantly surprised or taken by surprise, not both lol

File: news/2017-10-28-beatmap-spotlights-september-2017.md [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
++ **osu!taiko:** [Cryolien](https://osu.ppy.sh/u/1626983)
++ **osu!catch:** [\_Asriel](https://osu.ppy.sh/u/566276)
++ **osu!mania:** [Cryolien](https://osu.ppy.sh/u/1626983)
+
+
+## osu!
+
+[![](/wiki/shared/news/2017-10-28-beatmap-spotlights-september-2017/reloaded.jpg)](https://osu.ppy.sh/s/380143)
+
+[Handsome](https://osu.ppy.sh/u/2123087)'s latest ranked map, RELOADED, works most effectively thanks to its limited framework. The map is designed around a strict symmetrical object placement gimmick, along with the restriction of almost all sliders moving only upwards. Through this design, it provides an engaging experience that no other maps can provide!
```
Comment (Xinnoh) [inline news/2017-10-28-beatmap-spotlights-september-2017.md]: this is not handsome's latest ranked map

File: news/2017-10-28-beatmap-spotlights-september-2017.md [hunk 7]
```diff
@@ -0,0 +1,114 @@
...
+
+[![](/wiki/shared/news/2017-10-28-beatmap-spotlights-september-2017/we-are-one.jpg)](https://osu.ppy.sh/s/605745)
+
+In his latest mapset, [Frey](https://osu.ppy.sh/u/695678) presents us a fast-paced dubstep piece by renowned artist Pegboard Nerds. Five full difficulties of non-stop intense gameplay, all mapped fully by none other than himself. The very nature of the set brings us a genuine feeling of coherence and unity, a rare sight in the modern jungle of collaborations and guest difficulties.
```
Comment (Xinnoh) [inline news/2017-10-28-beatmap-spotlights-september-2017.md]: this is not frey's latest mapset

File: news/2017-10-28-beatmap-spotlights-september-2017.md (hunk 1 above)
Comment (Shadowthegod) [reply news/2017-10-28-beatmap-spotlights-september-2017.md]: Probably, but due to ProfessionalBox's long name its impossible to fit in. And I won't short his username either.

File: news/2017-10-28-beatmap-spotlights-september-2017.md [hunk 8]
```diff
@@ -0,0 +1,114 @@
+---
+layout: post
+title: "Beatmap Spotlights September 2017"
+date: 2017-10-28 16:00:00 +0000
```
Comment (Ephemeralis) [inline news/2017-10-28-beatmap-spotlights-september-2017.md]: Redate to today (in UTC)

File: news/2017-10-28-beatmap-spotlights-september-2017.md [hunk 9]
```diff
@@ -0,0 +1,114 @@
...
+---
+
+Here we are again, introducing a selection of the best maps of the previous month. Come take a look at September’s Beatmap Spotlights, and snag yourself a shiny new medal in the process!
```
Comment (Ephemeralis) [inline news/2017-10-28-beatmap-spotlights-september-2017.md]: make explicit reference to September since September isn't "last month" anymore

Comment (Shadowthegod) [reply news/2017-10-28-beatmap-spotlights-september-2017.md]: Did both, kinda unsure about the opener tho feels kinda meh to me now.

**Pull request merged**